packages/core/src/tools/instrumentMethod.ts

```diff
@@ -12,9 +12,12 @@ export function instrumentMethod<OBJECT extends { [key: string]: any }, METHOD e
 
   let instrumentation = instrumentationFactory(original)
 
-  const instrumentationWrapper = function (this: OBJECT): ReturnType<OBJECT[METHOD]> {
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-    return instrumentation.apply(this, arguments as unknown as Parameters<OBJECT[METHOD]>)
+  const instrumentationWrapper = function (this: OBJECT): ReturnType<OBJECT[METHOD]> | null {
+      if (typeof instrumentation === 'function') {
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+          return instrumentation.apply(this, arguments as unknown as Parameters<OBJECT[METHOD]>)
+      }
+      return null
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** any reason to return `null` here? I think it would be fine to return `undefined`. It's a bit subjective, since we don't know what could be returned, but explicitly returning `null` could raise some questions.

Reply:
`undefined` makes sense to me, I think I just got caught up on `null` from staring at errors. I also considered `void` with an empty `return` but I think `undefined` is probably more correct.